native/helpers/challenges.js

```diff
@@ -520,3 +520,12 @@ export const challenges = [
   { title: 'Emoji Editor', link: '#' },
   { title: 'Sort Table', link: '#' },
 ];
```

Comment:
There is a syntax error here. Please fix and check if the local server is running fine.

Reply:
Also arrange the challenge according to difficulty
Put it along with other easy difficulty challenges